chore: add missing data to lockfile

jsumners-nr opened the pull request:
## Proposed Release Notes

## Links

## Details

bizob2828 reviewed (commented):
do we want to change the ci to `npm ci` now?

bizob2828 commented:
yea we should update to prevent this issue in the future. here's the [workflow](https://github.com/newrelic/node-test-utilities/blob/main/.github/workflows/ci-workflow.yml#L40)

jsumners-nr commented:
@bizob2828 I have addressed your feedback, but I'm rather confident it will not solve the issue going forward. I believe the problem is `npm install <pkg>` isn't updating `package-lock.json` locally (could be due to a config I have on my system). 

What benefit does the `package-lock.json` provide us anyway? It doesn't get read when the module is installed because package managers ignore transitive lock files.
